### PR TITLE
Update troubleshooting.md

### PR DESCRIPTION
**Pull request by Goodiec**
This PR improves the troubleshooting doc and is part of [issue 41](https://github.com/jupyterhub/outreachy/issues/41)

### CODE REVIEW

File: docs/source/troubleshooting.md [hunk 1]
```diff
@@ -321,7 +312,7 @@ For instance:
     python3 -m pip install jupyterlab
     jupyter serverextension enable --py jupyterlab --sys-prefix
 
-The important thing is that jupyterlab is installed and enabled in the
+The important thing is that Jupyterlab is installed and enabled in the
```
Comment (minrk) [inline docs/source/troubleshooting.md:315]:
```suggestion
The important thing is that JupyterLab is installed and enabled in the
```

File: docs/source/troubleshooting.md [hunk 2]
```diff
@@ -1,32 +1,37 @@
 # Troubleshooting
 
 When troubleshooting, you may see unexpected behaviors or receive an error
-message. This section provide links for identifying the cause of the
+message. This section provides links for identifying the cause of the
 problem and how to resolve it.
 
 [_Behavior_](#behavior)
 
-- JupyterHub proxy fails to start
```
Comment (minrk) [inline docs/source/troubleshooting.md:9]: All these exact markdown links are sensitive to character changes in headings, and most [don't actually resolve](https://jupyterhub--4085.org.readthedocs.build/en/4085/troubleshooting.html).

Maybe we should remove all of these? They are redundant with the always-correct autogenerated TOC in the sidebar:

Screen Shot 2022-10-12 at 13 27 28

Comment (GeorgianaElena) [reply docs/source/troubleshooting.md:9]: @Goodiec, I believe what @minrk is suggesting is get rid of them entirely, including the bullet point lists.
They will still get autogenerated in the sidebar on the right.

Screenshot 2022-10-14 at 13 00 46

Comment (Goodiec) [reply docs/source/troubleshooting.md:9]: Got it, I'll update this shortly!

**Comment by minrk** [comment]
Wonderful, thank you!

**Pull request merged**